web/index.html

```diff
@@ -552,64 +601,147 @@ <h3>Used in</h3>
                     };
 
                     const animateScroll = (currentTime) => {
+                        if (this.isScrolling) {
+                            return;
+                        }
```

Comment:
If the user is scrolling at all, just instantly cancel, this helps not cause the user to ever be fighting with the automated scrolling